composer.json

```diff
@@ -54,7 +54,9 @@
         "vimeo/psalm": "^4.0"
     },
     "suggest": {
-        "ext-inotify": "To use inotify based file watcher. Required for hot code reloading."
+        "ext-inotify": "To use inotify based file watcher. Required for hot code reloading.",
+        "ext-openswoole": "The package requires at least one of OpenSwoole or Swoole",
+        "ext-swoole": "The package requires at least one of OpenSwoole or Swoole"
```

Comment:
The trademark is "OpenSwoole" but everywhere it is written: "Open Swoole".

* https://www.swoole.co.uk
* https://github.com/openswoole
* https://github.com/mezzio/mezzio-swoole/pull/73/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R6

Reply:
I'm assuming, then, that when I refer to it in a written fashion that it should be "Open Swoole" everywhere, based on that. I've updated the patch to reflect that.